src/mozanalysis/experiment.py

```diff
@@ -409,10 +421,14 @@ def build_metrics_query(
         )
 
         return """
+        WITH enrollments AS (
+            SELECT *
+            FROM {enrollments_table}
+        )
         SELECT
-            {enrollments_table}.*,
+            enrollments.*,
             {metrics_columns}
-        FROM {enrollments_table}
+        FROM enrollments
```

Comment:
- We should write `` `{enrollments_table}` `` to make sure it's escaped, right?
- The CTE seems more complex than writing ``FROM `{enrollments_table}` AS enrollments``

Reply:
> The CTE seems more complex than writing `FROM `{enrollments_table}` AS enrollments`

Yeah that's a much neater solution...

Reply:
Huh, this appears not to work with the data source subquery in the left join - maybe I tried this originally last year and switched to the CTE accordingly. Back to the ugly version then, until BQ supports the neater one...